src/screens/NavigationComponent.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import { useAccountState } from "../contexts/accountContext";
+
+import DrawerScreen from "./DrawerScreen";
+import UnauthenticatedStackScreen from "./UnauthenticatedStackScreen";
+
+const NavigationComponent = () => {
+  const account = useAccountState();
+  return (
+    <>
+      {account.account.email === "" ? (
+        <UnauthenticatedStackScreen />
```

Comment:
Good to see that you got this type of routing implemented!

Reply:
yes It also automatically will route once an account logs out saving us some code